tests/lib/mountinfo-tool

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
+from abc import ABC, abstractmethod
+from argparse import ArgumentParser, FileType, RawTextHelpFormatter
+from typing import Dict, List, Tuple, Any, Type, Callable, Match
+import re
+
+
+class Device(int):
+    """
+    Device is a device number with major and minor components.
+
+    Note that this class does not attempt to mimic peculiar
+    encoding used by the Linux kernel.
+    """
+
+    @classmethod
+    def pack(cls, major: int, minor: int) -> 'Device':
+        return cls((major << 16) | (minor & (1 << 16) - 1))
```

Comment:
Why going back to int if you're handled a `(maj, min)` tuple already? Would `Device(Tuple[int,int])` work?

Reply:
Because `Device` inherits from `int`. It is not an object with a dict but a real int with custom methods.